examples/lighting-app/nrfconnect/CMakeLists.txt

```diff
@@ -30,16 +30,32 @@ if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/boards/${BOARD}.conf)
   list(APPEND CONF_FILE boards/${BOARD}.conf)
 endif()
 
-option(BUILD_WITH_DFU "Build target with Device Firmware Upgrade support" OFF)
-if(BUILD_WITH_DFU)
+set(BUILD_WITH_DFU "MATTER" CACHE STRING "Build target with Device Firmware Upgrade support")
+if(NOT ${BUILD_WITH_DFU} STREQUAL "OFF")
```

Comment:
In theory, one could want both DFU method at once. How about having two flags: DFU_BLE=y/n and DFU_MATTER=y/n?

Reply:
If you set "BLE" option it will result in enabling both Matter and BLE DFUs, look at the code below. I didn't create an option to build DFU over BLE only, as I thought that is what we assumed withing this task.

Reply:
What it currently works is that not putting any argument (so the default "MATTER") enables OTA Matter and "BLE" enables OTA Matter + BLE

Reply:
Ok, I get it now.